GitUI/BranchTreePanel/ContextMenu/GitRefsSortOrderContextMenuItem.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Linq;
+using System.Windows.Forms;
+using GitCommands;
+using GitCommands.Utils;
+using GitUI.Properties;
+using GitUIPluginInterfaces;
+using ResourceManager;
+
+namespace GitUI.BranchTreePanel.ContextMenu
+{
+    internal class GitRefsSortOrderContextMenuItem : ToolStripMenuItem
+    {
+        internal const string MenuItemName = "GitRefsSortOrderContextMenuItem";
+        private readonly TranslationString _sortByText = new TranslationString("&Sort order...");
```

Comment:
translate?

Reply:
It appears these aren't translatable in the current form. 
I hoped no-one would notice :D